pages/about.html

```diff
@@ -1,23 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>About</title>
+<title>Mount Sion CBS Secondary School</title>
     
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
-  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js"></link>
-  <script src="//code.jquery.com/jquery-2.2.1.js"></script>
+    <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js">
+   <script src="//code.jquery.com/jquery-2.2.1.js"></script>
     
 </head>
  <body>
     
-<ul>
-  <li><a class="active" href="about.html">About</a></li>
-  <li><a href="index.html">Home</a></li>
-  <li><a href="history.html">History</a></li>
-  <li><a href="contact.html">Contact</a></li>
-</ul> 
-    
-    
-    
+<img class="ui centered medium image" src="https://i.imgur.com/ajcswUA.jpg">
```

Comment:
Just an idea, It might be worth wrapping the menu and image in a container, this should help keep the page clean moving forward. `<div class=ui container>` should be sufficient for this. Only a suggestion, I would like to see what other people think on this.